index.html

```diff
@@ -29,20 +35,27 @@
   <p>
     Several of our community coordinators have contact email addresses for players who prefer to connect
     via email.
-    <ul>{% for contact in site.data.contacts %}
-      <li><a href="mailto:{{ contact.email }}">{{ contact.agent }}</a> ({{ contact.area }}) {% if contact.form != nil %} <a href="{{contact.form}}">Sign-up form</a>{% endif %}</li>{% endfor %}
+    <ul>
+        {% assign contacts = site.data.contacts | sort: 'area' %}
+        {% for contact in contacts %}
+        <li>
+            <b>{{ contact.area }}</b>
```

Comment:
I think that getting a redirect to a community is best, and the agents are sort-of "last case", at least the way the current language is written.